### ISSUE TITLE
Plots display floating point errors

### ISSUE THREAD

**Issue by ianfiske** [JuliaPlots/Plots.jl#2079]
I'm having trouble getting plots to stop showing floating point errors:

```
using Plots

plot([0.2, 0.4, 0.6, 0.8])
```

![image](https://user-images.githubusercontent.com/135570/60371752-2f71b200-99c8-11e9-9e8c-8bacb60d56d4.png)

Trying explicitly to format helps somewhat:

```
plot([0.2, 0.4, 0.6, 0.8], yformatter = x -> string(x))
```

![image](https://user-images.githubusercontent.com/135570/60371788-529c6180-99c8-11e9-9f6d-9702f34b62df.png)

Is this a bug or is there a simple fix I'm missing?  I'm on Plots v0.25.2.  Thanks!

**Comment by AshtonSBradley** [comment]
I am getting the same thing - new behaviour that must have developed during an update in the last few days:
```julia
using Plots
n(x) = x^2/(1+x^2)
x = LinRange(0,3,50)
plot(x,n.(x))
```
gives 

<img width="411" alt="Screen Shot 2019-06-30 at 5 32 27 AM" src="https://user-images.githubusercontent.com/24194258/60387520-99897480-9af8-11e9-8278-8655a1cfa325.png">

```julia 
[91a5bcdd] Plots v0.25.2

julia> versioninfo()
Julia Version 1.1.1
Commit 55e36cc (2019-05-16 04:10 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin15.6.0)
  CPU: Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 4
```

**Comment by lobingera** [comment]
Showoff.jl got updated recently.

**Comment by jheinen** [comment]
Can confirm that's it is related to the `Showoff.jl` v0.2.1 => v0.3.0 update.

**Comment by daschw** [comment]
Thanks all for reporting, this was my fault, should be fixed with https://github.com/JuliaGraphics/Showoff.jl/pull/17

**Comment by daschw** [comment]
Fixed with Showoff v0.3.1 https://github.com/JuliaRegistries/General/pull/1671
